README.md

```diff
@@ -19,6 +19,7 @@ A [Sourcegraph extension](https://docs.sourcegraph.com/extensions) for showing c
 -   Line branches/hits annotations on files
 -   File coverage ratio indicator (`Coverage: N%`) and toggle button
 -   Support for using a Codecov API token to see coverage for private repositories
+-   File and directory coverage decorations on Sourcegraph.com
```

Comment:
The first point here is "Support for GitHub.com and Sourcegraph.com", so I believe this is trying to communicate that it works on the web app. By only listing Sourcegraph.com, I'm trying to communicate that we haven't implemented file decorations for code hosts yet. However, this could be confusing when you consider private instances. Should we remove ".com" from the first point too?